asciidoc-confluence-publisher-cli/src/main/java/org/sahli/asciidoc/confluence/publisher/cli/AsciidocConfluencePublisherCommandLineClient.java

```diff
@@ -64,6 +68,7 @@ public static void main(String[] args) throws Exception {
 
             AsciidocConfluenceConverter asciidocConfluenceConverter = new AsciidocConfluenceConverter(spaceKey, ancestorId);
             ConfluencePublisherMetadata confluencePublisherMetadata = asciidocConfluenceConverter.convert(asciidocPagesStructureProvider, pageTitlePostProcessor, buildFolder);
+            confluencePublisherMetadata.setPublishStrategy(publishStrategy);
```

Comment:
Instead of setting the publish strategy onto the `ConfluencePublisherMetadata`, you could just pass the publish strategy to the `ConfluencePublisher` constructor - the `ConfluencePublisherMetadata` does not need to know about the publish strategy (as this is a responsibility of the `ConfluencePublisher`)